namespaces/live.cloud-platform.service.justice.gov.uk/use-of-force-dev/resources/analytical-platform-access.tf

```diff
@@ -66,7 +66,7 @@ resource "kubernetes_secret" "ap_aws_secret" {
   }
 
   data = {
-    destination_bucket = "s3://hmpps-${var.namespace}-landing"
+    destination_bucket = "s3://moj-reg-dev/landing/us-of-force-dev/"
```

Comment:
Is this S3 bucket already created and named? Just a query with the name?

`us-of-force-dev/` should it be `use-of-force-dev/`

Reply:
yes, this bucket is created, fixed the type